nss-day-cohort-39/tabloid-full-stack-velours
# debug category delete

### Server-Side Changes
- Added IsDeleted boolean to Category.cs
- Removed Delete() method and GetById() method from CategoryRepo.cs
- Removed [HttpDelete] from CategoryController.cs
- Ran SQL Query to insert IsDeleted column into Category table

### Client-Side Changes
- Deleted deleteCategory() function from CategoryProvider.js
- Added isDeleted property to category object in editCategory() function in Category.js
- Added deleteCategory() function to Category.js to be called onClick
- Added (category.isDeleted === false) && to Category.js to filter for display categories that aren't deleted

### Testing
- Run SQL code in spreadsheet to add IsDeleted column to Category table
- Execute VS solution and `npm start` in client
- Login as admin and click "Categories" in Header
- Delete category and make sure it disappears, page re-renders, and IsDeleted changes to true in SQL db
- Make sure prior functionality from recent master also still works
